.github/workflows/test.yml

```diff
@@ -67,3 +67,42 @@ jobs:
           FOLDER: D3DB
           SKIP_EMPTY_COMMITS: true
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  build-website:
+    name: Run D3 Build of website for github pages
+    environment:
+      name: d3db-gh-pages
+      url: https://github.com/TechWorksHub/ManySecured-D3DB/tree/gh-pages
+    concurrency: d3db-gh-pages # limit to a single-run build at a time
```

Comment:
This should be in the `deploy-website` job, not the `build-website` job.

Reply:
Oh you already have an `environment` in `deploy-website`. In that case, you should delete this one, since this job doesn't actually deploy anything, it just builds the website.

```suggestion
```